driver/objgen/objgen_test.go

```diff
@@ -20,3 +22,31 @@ func Test_Instantination(t *testing.T) {
 	require.Nil(t, err)
 	require.Zero(t, len(res.UnstructuredObjects))
 }
+
+func TestGetTemplate(t *testing.T) {
```

Comment:
I have a better idea to cut out the secrets from output.
here the output type definition
```go
type Output struct {
	RawYaml             []byte
	UnstructuredObjects []*unstructured.Unstructured
	//Not yet ready
	//DaemonSet           []*appsv1.DaemonSet
	//ClusterRoleBinding  []*rbacv1.ClusterRoleBinding
	//ClusterRole         []*rbacv1.ClusterRole
	//ServiceAccount      []*v1.ServiceAccount
	//ConfigMap           []*v1.ConfigMap
	//Secret              []*v1.Secret
}
```

We just need to:
1. add parsing the yaml to objects (I was a bit lazy to do it initially, and just return the raw yaml), so except of the populating `RawYaml` attribute we also populate all of the objects slices
2. add serialize method to `Output` type with some parameters (allowing to skip what ever you want to skip), so this method can provide you with the final raw yaml. Thus we can remove the `RawYaml` attribute at all from the `Output` type.

Reply:
Yep that sounds much cleaner. I wasn't sure what was the status of implementing those fields so I thought to go other route for now.

Reply:
@i-prudnikov What do think. Should I implement those now in this PR or is this something that we want to maybe refactor in separate PR or do first in CAPU or CCM ?

Reply:
I think it is ok to do it here. In CAPU it is already a manifest that is parsed to objects, but it still lack of serialize method to `Output` type  that I was hoping to copy and paste from you, once you've implemented this.

Reply:
In CCM we have exactly the same state as in CSI for now.

Reply:
OK. I'll start working on it.